pkg/sloop/common/db_utilities.go

```diff
@@ -5,8 +5,8 @@ import (
 	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
 )
 
-func deleteKeys(db badgerwrap.DB, keysForDelete [][]byte) (error, int) {
-	deletedKeysInThisBatch := 0
+func deleteKeys(db badgerwrap.DB, keysForDelete [][]byte) (error, uint64) {
+	var deletedKeysInThisBatch uint64 = 0
```

Comment:
any reason we change from int to uint64?

Reply:
the number of keys cannot be negative so changed it uint64 like other places in code